### PR TITLE
Add desktop form for recording videos

### PR DESCRIPTION
**Pull request by vasylmrnchk**

### CODE REVIEW

**Review by Anatolii-Ivanov — COMMENTED** [review]
Can you add a screenshot of the result?

**Comment by vasylmrnchk** [comment]
> 
formscreen
yes, of course

File: RecodoDesktop/RecodoDesktop/VideoRecordingForm.xaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        </Button>
+
+        <Button Grid.Column="1" Grid.Row="1" Style="{StaticResource MaterialDesignFlatAccentButton}" HorizontalAlignment="Right" VerticalAlignment="Top" Margin="0,0,27.32,0" Height="36" Width="198">
+            <StackPanel Orientation="Horizontal">
+                <Image
+                    Height="15"
+                    Width="24"
+                    Source="Icons/image.png"/>
+                <TextBlock
+                    Text="Screenshot (Beta)"
```
Comment (Jyuart) [inline RecodoDesktop/RecodoDesktop/VideoRecordingForm.xaml]:
```suggestion
                    Text="Screenshot (coming soon)"
```

To be more clear

File: RecodoDesktop/RecodoDesktop/VideoRecordingForm.xaml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        <Button Grid.Row="1" Grid.Column="0" Margin="34.62,0,0,11.13" Height="36" Padding="0,4,16,4" HorizontalContentAlignment="Stretch" Width="198" Style="{StaticResource MaterialDesignRaisedSecondaryLightButton}">
+            <Button.BorderBrush>
+                <SolidColorBrush Color="#4CE0748B" Opacity="30"/>
+            </Button.BorderBrush>
+            <Button.Background>
+                <SolidColorBrush Color="#4CE0748B" Opacity="30"/>
+            </Button.Background>
+            <StackPanel Orientation="Horizontal" HorizontalAlignment="Left" VerticalAlignment="Center" Margin="33.21,0,0,0">
+                <Image
+                    Height="15"
```
Comment (Jyuart) [inline RecodoDesktop/RecodoDesktop/VideoRecordingForm.xaml]: Please, fix formatting here and in other places below to make it consistent

**Review by Anatolii-Ivanov — DISMISSED** [review]

**Comment by vasylmrnchk** [comment]
rounded-form

**Comment by Jyuart** [comment]
> rounded-form

Just one thing. 
image
Can you move those ticks a little bit left? I can't imagine QAs don't notice it

**Comment by vasylmrnchk** [comment]
> 
i'm working on it now

**Comment by vasylmrnchk** [comment]
new-arrows-position

**Review by Anatolii-Ivanov — DISMISSED** [review]

**Review by Jyuart — DISMISSED** [review]

**Comment by aydevelop** [comment]
Perhaps too close

**Comment by vasylmrnchk** [comment]
> Perhaps too close 

I`m fix it, like that
Без імені

**Review by Jyuart — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**